en/development/exceptions.rst

```diff
@@ -64,7 +64,7 @@ exceptions for HTTP methods
 
 .. php:exception::UnauthorizedException
```

Comment:
@blackpuppy also need a space here `.. php:exception:: UnauthorizedException`.
I added in ff8abcf6597cef2765ed5601f825ef6b424bde23 but it's just to let you know for your translation